Formula/waybackpy.rb

```diff
@@ -0,0 +1,51 @@
+class Waybackpy < Formula
+  include Language::Python::Virtualenv
+
+  desc "Wayback Machine API interface & command-line tool"
+  homepage "https://pypi.org/project/waybackpy/"
+  url "https://files.pythonhosted.org/packages/34/ab/90085feb81e7fad7d00c736f98e74ec315159ebef2180a77c85a06b2f0aa/waybackpy-3.0.6.tar.gz"
+  sha256 "497a371756aba7644eb7ada0ebd4edb15cb8c53bc134cc973bf023a12caff83f"
+  license "MIT"
+
+  depends_on "python@3.10"
```

Comment:
why not `python@3.11`?

Reply:
Apologies, I accidentally deleted my comments. I’ve added explicit 3.11 support to upstream https://github.com/akamhy/waybackpy/pull/179 but I would rather wait for a new release there before I target 3.11.